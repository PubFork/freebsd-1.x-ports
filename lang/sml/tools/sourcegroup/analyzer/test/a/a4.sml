signature A = sig
  structure B :sig val x:int end
  structure D :sig val y:int end
end


(* {dst B dst D}*(dsi A [dst B dst D]) *)
