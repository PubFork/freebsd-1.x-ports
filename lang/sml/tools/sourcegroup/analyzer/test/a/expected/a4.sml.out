source sml "a/a4.sml"
  export signature A ;

